src/bull.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const { randomNumber } = require('./getRandomNumber');
+
+function bull(userInput) {
+  let count = 0;
+
+  const arrInput = userInput.split('');
+  const arrNumber = randomNumber.split('');
```

Comment:
```suggestion
```
Useless code. randomNumber has length and includes() method. Also you can get chars by indexes. So you don;t need arrays at all.